skptesting/slow-calls-multi-proxy-lbroutes.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+
+# build go test binaries
+go build -o gohttpclient gohttpclient.go
+#go build -o gorilla-ws-example gorilla-ws-example.go
+
+
+# shellcheck source=./benchmark.inc
+source "$GOPATH/src/github.com/zalando/skipper/skptesting/benchmark.inc"
+
+if [ "$1" == -help ] || [ "$1" == --help ] || [ "$1" == -h ]; then
+	log "$0 [duration] [connections] [warmup-duration]"
+	exit 0
+fi
+
+check_deps
+
+trap cleanup-exit SIGINT
+
+log; log [starting servers]
+
+# 2 backends
+skipper -experimental-upgrade -access-log-disabled -address ":9000" -inline-routes "r: * -> inlineContent(\"CORRECT\") -> status(200) -> <shunt>;" &
+pids="$pids $!"
+skipper -experimental-upgrade -access-log-disabled -address ":9001" -inline-routes "r: * -> inlineContent(\"WRONG\") -> status(200) -> <shunt>;" &
+pids="$pids $!"
+
+# lb
+skipper -experimental-upgrade -address :9091 -proxy-preserve-host -routes-file "$cwd"/lb_wrong.eskip \
+	-idle-conns-num 2 -close-idle-conns-period 3 &
+pids="$pids $!"
+
+# fake alb
+skipper -experimental-upgrade -access-log-disabled -proxy-preserve-host -idle-conns-num 20 -close-idle-conns-period 30 -inline-routes 'r: * -> "http://127.0.0.1:9091";' &
+pids="$pids $!"
+log [servers started, wait 1 sec]
+
+sleep 1
+
+#log; log [start connections wrk]
+#wrk -H "Host: test.example.org"  -c "5" -d "12s" http://127.0.0.1:9090/  &
+
+SEC=8
+log; log [start connections vegeta]
```

Comment:
as i see we don't use vegeta here, but the custom client, change vegeta -> client

Reply:
done